src/translations/en_SE.json

```
@@ -66,7 +66,7 @@
   "CHECKOUT_EMAIL_PLACEHOLDER": "your.email@here.com",
   "CHECKOUT_SUMMARY_TITLE": "Your details",
   "CHECKOUT_SIGN_BUTTON_TEXT": "Sign with mobile BankID",
-  "CHECKOUT_SIGN_DISCLAIMER": "By clicking “Sign with BankID” I confirm that I have read and understood [the pre-sale information]({PREBUY_LINK}), [terms]({TERMS_LINK}) and that my personal data is handled [according to GDPR](https:\/\/www.hedvig.com\/privacy).",
+  "CHECKOUT_SIGN_DISCLAIMER": "By signing I confirm that I have read and understood the [{TERMS_TITLE}]({TERMS_LINK}) and the [{IPID_TITLE}]({PREBUY_LINK}), and that Hedvig handles my [personal information](https:\/\/www.hedvig.com\/privacy).",
```

As I said we can omit the copy details and see what Sara thinks, personally I think it's a neater sentence without double _and_ in the english version but it makes it less obvious with the links though 
```suggestion
  "CHECKOUT_SIGN_DISCLAIMER": "By signing I confirm that I have read and understood the [{TERMS_TITLE}]({TERMS_LINK}), [{IPID_TITLE}]({PREBUY_LINK}) and that Hedvig handles my [personal information](https:\/\/www.hedvig.com\/privacy).",
```

---

Actually I think it's better the way I wrote it 😄 The way you wrote it the sentence becomes a bit strange since that means that we say **"By signing I confirm that I have read and understood that Hedvig handles my personal information"**. **"I have read that Hedvig handles my personal information"** does not seem right?

If you don't like the "and" we should rather split it up into two sentences imo:
```
"By signing I confirm that I have read and understood the [{TERMS_TITLE}]({TERMS_LINK}) and the [{IPID_TITLE}]({PREBUY_LINK}). 
I also approve that Hedvig handles my [personal information](https:\/\/www.hedvig.com\/privacy)."
```

---

Yes that looks weird! :D No let's keep it! And we know we will change the links later on 🙌